src/client/components/pages/collection.js

```diff
@@ -74,6 +74,10 @@ function CollectionAttributes({collection}) {
 					<dt>Collection type</dt>
 					<dd>{collection.entityType}</dd>
 				</Col>
+				<Col md={3}>
+					<dt>No. of Entity</dt>
```

Comment:
I would suggest this: 
```suggestion
					<dt>Number of entities</dt>
```